config/ie/postCoreHoudiniInstall

```diff
@@ -42,7 +42,7 @@
 import sys
 
 import VersionControl 
-VersionControl.setVersion('IEBuild', '6.15.0') 
+VersionControl.setVersion('IEBuild', '6') 
```

Comment:
It's mainly convention to use an exact version of IEBuild, but we don't actually install the major verison symlink, so that convention is enforced currently. I think we can just change this to 6.23.1 once you've done that release.